src/test_typing_extensions.py

```diff
@@ -5805,6 +5806,11 @@ class MyTypedDictT(Generic[T], TypedDict):
             self.assertEqual(typing_extensions._collect_parameters((Unpack[MyTypedDictT[T]], )), (T,))
             self.assertEqual(typing_extensions._collect_parameters((Unpack[Tuple[int]], )), ())
             self.assertEqual(typing_extensions._collect_parameters((Unpack[Tuple[int, T]], )), (T,))
+        self.assertEqual(typing._GenericAlias(Any, Unpack[MyTypedDict]).__parameters__, ())
```

Comment:
This is still a private object that shouldn't be used directly. Can you focus the tests on use of public functionality that currently doesn't work and is fixed by your change?

Reply:
I think I then misunderstood what you meant by "keep the tests here" 🙃 . Can you clarify that for me?

I interpreted "here" as in the code, but it should have some other case added. Do I then interpret it correctly that you mean "here" with here in this PR and I should remove this whole block for the private function?

Reply:
Thinking about it more: Let's have this PR add only tests that exercise public functionality, and remove any tests that test only private functions.